src/test/java/org/ays/payload/ApplicationStatus.java

```diff
@@ -0,0 +1,8 @@
+package org.ays.payload;
+
+public enum ApplicationStatus {
```

Comment:
Aşağıdaki şekilde güncellememiz daha iyi olur, ilerleyen dönemde başka başvuru aşamaları da olacak, karışabilir
```suggestion
public enum AdminUserRegisterationApplicationStatus {
```

Reply:
Haklisin @agitrubard. Değiştirip yolluyorum.